src/ocispec/ocijsonspec.hpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef OCIJSONSPEC_HPP_
+#define OCIJSONSPEC_HPP_
+
+#include <zephyr/data/json.h>
+
+#include <aos/common/ocispec.hpp>
+
+static constexpr size_t cMaxDeviceTreeLength = 1024;
```

Comment:
Where is it used?

Reply:
aos::StaticBuffer<cJsonMaxContentSize> mJsonFileBuffer;

Reply:
cJsonMaxContentSize used there. I don't see this const is used anywhere.